python_ta/typecheck/type_store.py

```diff
@@ -1,12 +1,13 @@
 import astroid
 from collections import defaultdict
 from typing import Callable, Any
-
+import os
+TYPE_SHED_PATH = os.path.join(os.path.dirname(__file__), 'typeshed/builtins.pyi')
```

Comment:
Don't hard-code the forward-slash (different on Windows). Simply pass `'typeshed'` and `'builtins.pyi'` as arguments to the `os.path.join` -- it'll join them properly.